lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
@christian-bromann if you remove `this.elements` you don't get a selector for the situations like

`this.client.element('[name="login"]').hasFocus()`

you get `null`